cms/static/js/views/paging_header.js

```diff
@@ -23,6 +28,7 @@ define(["underscore", "backbone", "gettext", "text!templates/paging-header.under
                     lastPage = collection.getTotalPages(),
                     messageHtml = this.messageHtml(),
                     isNextDisabled = lastPage === 0 || currentPage === lastPage;
+
                 this.$el.html(_.template(paging_header_template)({ messageHtml: messageHtml}));
```

Comment:
Use ``HtmlUtils.setHtml`` and ``HtmlUtils.template`` here to safely manage HTML snippets.